File: ImageSingle.Droid/MainActivity.fs
namespace ImageSingle.Droid

open System
open Android.App;
open Android.Content;
open Android.Content.PM;
open Android.Runtime;
open Android.Views;
open Android.Widget;
open Android.OS;

type Resources = ImageSingle.Droid.Resource

[<Activity (Label = "ImageSingle", MainLauncher = true, Icon = "@mipmap/icon")>]
type MainActivity() =
    inherit Xamarin.Forms.Platform.Android.FormsApplicationActivity()
    override this.OnCreate (bundle: Bundle) =
        base.OnCreate (bundle)

        Console.WriteLine("activity is starting")

        Xamarin.Forms.Forms.Init (this, bundle)
        this.LoadApplication (new ImageSingle.App ())

File: ImageSingle.DroidCSharp/MainActivity.cs
using Android.App;
using Android.Widget;
using Android.OS;

namespace ImageSingle.DroidCSharp
{
    [Activity(Label = "ImageSingle.DroidCSharp", MainLauncher = true, Icon = "@mipmap/icon")]
    public class MainActivity : Xamarin.Forms.Platform.Android.FormsApplicationActivity
    {
        protected override void OnCreate(Bundle savedInstanceState)
        {
            base.OnCreate(savedInstanceState);

            Xamarin.Forms.Forms.Init(this, savedInstanceState);
            this.LoadApplication(new ImageSingle.App());
        }
    }
}

